src/ln/channelmanager.rs

```diff
@@ -2567,10 +2574,27 @@ impl ChainListener for ChannelManager {
 				}
 				true
 			});
+			channel_state.claimable_htlcs.retain(|payment_hash, htlcs| {
+				htlcs.retain(|htlc| {
+					if htlc.cltv_expiry <= height { // XXX: Or <?
```

Comment:
Sorry what means "XXX: or <?" ? You want to substract a safe delay  to height ?

Reply:
Oops, lol, ffs, I really need to double-check my own commits before opening PRs, thanks for reviewing...That was a note to myself that I'm not sure if the comparison was supposed to be <= or <, I'll double check and fix the PR.